client/src/components/Header/styles.tsx

```diff
@@ -1,13 +1,10 @@
 import { makeStyles, Theme } from '@material-ui/core/styles';
 
 export default makeStyles((theme: Theme) => ({
-  header: {
-    color: theme.palette.primary.main,
-    height: '4em',
-    padding: '0.25em 0.25em',
-    display: 'flex',
-    alignItems: 'center',
-    backgroundColor: theme.palette.common.white,
+  appbar: {
+    // Necessary to override default z-index value of appbar
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment

```

Comment:
It looks like a different/additional rule needs disabling here.

Reply:
There are a couple of lint rules that get chained into each other when disabled; I've gone ahead and just disabled them for the entire file since the next line disabling doesn't work here.